src/main/java/com/nha/abdm/wrapper/hrp/common/MakeRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.nha.abdm.wrapper.hrp.common;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+
+public class MakeRequest {
+
+    public static ResponseEntity<ObjectNode> post(String uri, HttpEntity<ObjectNode>requestEntity) {
+        WebClient.Builder webClientBuilder = WebClient.builder();
```

Comment:
For best performance, the `WebClient` should be built once and and reused for all requests, as it can internally manage the http connections.

Reply:
Understood and modified the code to make the WebClient built only once.